src/schema/sources.ts

```diff
@@ -805,7 +806,11 @@ export const resolvers: IResolvers<any, Context> = {
         .getRepository(Source)
         .findOneBy({ handle: handle.toLowerCase() });
 
-      return !!source;
+      const disallowHandle = await ctx
```

Comment:
Can ber merged we the above using `Promise.all`

Reply:
Also why don't you use `checkDissalowHandle`?
We can add the `toLowerCase` to the function as well to keep it consistent